src/bvals/comms/bnd_info.cpp

```diff
@@ -40,8 +40,9 @@
 namespace parthenon {
 
 void ProResCache_t::Initialize(int n_regions, StateDescriptor *pkg) {
-  prores_info = ParArray1D<ProResInfo>("prores_info", n_regions);
-  prores_info_h = Kokkos::create_mirror_view(prores_info);
+  prores_info = ProResInfoArr_t(ViewOfViewAlloc("prores_info"), n_regions);
+  prores_info_h = Kokkos::create_mirror_view(
+      Kokkos::view_alloc(Kokkos::SequentialHostInit), prores_info);
```

Comment:
This piece of the mirror discussion in addition to `ViewOfViewAlloc` might be worth describing in our docs note.

Reply:
Which piece are you referring to?
There's already sth on the `view_alloc`in the current docs.

Reply:
ah I missed that. We're good then. :+1: